maps/groundbase/groundbase_telecomms.dm

```diff
@@ -73,9 +73,8 @@
 		num2text(SEC_I_FREQ)=list(access_security),
 		num2text(SCI_FREQ) = list(access_tox,access_robotics,access_xenobiology),
 		num2text(SUP_FREQ) = list(access_cargo),
-		num2text(SRV_FREQ) = list(access_janitor, access_hydroponics),
-		num2text(EXP_FREQ) = list(access_explorer)
-	)
+		num2text(SRV_FREQ) = list(access_janitor, access_hydroponics)
+		)
```

Comment:
I think Exploration frequency should remain, albeit renamed to a more generalized "Volunteer" channel.

It is very useful when you got medical, engineering, science and journo volunteers to just yeet the explo headsets at them because otherwise, SD people freak out when they hear away team chatter.